.gitignore

```diff
@@ -15,3 +15,4 @@ output.txt
 **/katana-logs
 crates/benches/bench_results.txt
 **/generated
+.vscode/settings.json
```

Comment:
i think we can ignore all vscode related stuff with `.vscode/**`

Reply:
I'm very new to use .vscode like editor, and I've seen some `launch.json` file. Not sure how important it is.
But if it's usually all ignored, let's do it.